values.yaml

```diff
@@ -45,16 +53,14 @@ minio:
   ingress:
     enabled: true
     hostname: data.outline.yourdomain.tld
-    certManager: false
     tls: false
     annotations:
       # cert-manager.io/cluster-issuer: "letsencrypt-staging"
   mode: "standalone"
   defaultBuckets: "ol-data" # should not be "outline" because of https://github.com/outline/outline/issues/2068
-  secretKey:
-    password: "Uk4LuwVFvRDiG6k4nYA5C^835V" # MUST be replaced with some custom generated secret! Don't use this in production
-  accessKey:
-    password: "zFPVNNK*QNp3^V4J38Gy76SehG" # MUST be replaced with some custom generated access_key! Don't use this in production
+  auth:
```

Comment:
This breaking change is provided because of the minio upgrade? This should be documented in README.md to avoid breaking upgrades.

Reply:
Added to readme